packages/world/ts/exports/internal.ts

```diff
@@ -11,3 +11,5 @@ export * from "../encodeSystemCallsFrom";
 export * from "../actions/callFrom";
 
 export * from "../callWithSignatureTypes";
+
+export { resolveTableId, resolveWithContext } from "../config/v2/dynamicResolution";
```

Comment:
I am intentionally exporting `resolveTableId` in the internal bundle for now as I want to reconsider the API before we make it concrete.